cobc/parser.y

```diff
@@ -12253,6 +12253,20 @@ call_body:
 
 	/* Check parameter conformance, if we can work out what is being called. */
 	if (CB_LITERAL_P ($3)) {
+		/* Check for "Java." prefix and set call convention */
+		char* s = (char *)CB_LITERAL ($3)->data;
+		if (strncasecmp("Java.", s, 5) == 0) {
+			call_conv = CB_CONV_JAVA;
+		}
+		/* Check for malformed Java method names */
+		char* class_and_method_name = s + 5;
```

Comment:
You can probably avoid this intermediate declaration (which is not at the beginning of a block), and just use `s + 5` in `strchr` instead.